test/features/support/hooks.rb

```diff
@@ -1,5 +1,12 @@
 Before do
   Mongoid.purge!
+  on(LandingPage).load
+end
+
+Before('@test_user') do
+  user = User.new
```

Comment:
:+1: I think I just lifted from what it was doing, but I think that's a better thing to do.